resources/framework.rb

```diff
@@ -21,7 +21,8 @@
 #
 
 # The provides method is available on chef >= 12.0 only
-provides :ms_dotnet_framework, os: 'windows' if respond_to?(:provides)
+provides :ms_dotnet_framework, os: 'windows'
+unified_mode true if respond_to?(:provides)
```

Comment:
I think you mean `if respond_to?(:unified_mode)`

Reply:
Yup, fixed. Odd that happened since most of the changes here were generated by using autocorrect with Cookstyle